sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -194,6 +196,17 @@ export class StudentService extends RecordDataModelService<Student> {
       throw error;
     }
 
+    // If sfasIndividual wasn't found, check for a partial match
+    if (!sfasIndividual) {
+      this.getStudentPartialMatch(
```

Comment:
The `get` prefix can be misleading since the method executes a check to possibly generate a notification.
Can we have it renamed to `checkLegacyIndividualPartialMatch`?

Reply:
updated